app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,63 @@
+class FriendshipsController < ApplicationController
+  @is_unfriend = false
+  def new
+    @users = User.all_except(current_user).order(:name).page(params[:page])
+  end
+
+  def create
+    @friendship = current_user.friendships.new(friend_id: params[:friend_id])
+    if @friendship.save
+      flash[:success] = "Request sent successfully"
```

Comment:
only `flash[:success] = "Request sent successfully"` should be in if else block.
`redirect_to new_friendship_path` should be outside if this block.